Source/WebCore/rendering/style/RenderStyle.cpp

```diff
@@ -2891,6 +2892,16 @@ void RenderStyle::setFontSizeAdjust(FontSizeAdjust sizeAdjust)
     fontCascade().update(selector);
 }
 
+void RenderStyle::setFontOpticalSizing(FontOpticalSizing opticalSizing)
+{
+    auto selector = fontCascade().fontSelector();
+    auto description = fontDescription();
+    description.setOpticalSizing(opticalSizing);
+
+    setFontDescription(WTFMove(description));
+    fontCascade().update(selector);
```

Comment:
I see this is copying the prevailing style of other setters on `RenderStyle`, but the raw pointer `selector` makes me queasy.